tools/expotools/src/Changelogs.ts

```diff
@@ -157,131 +171,137 @@ export class Changelog {
         continue;
       }
 
-      if (currentVersion && currentSection && token.type === Markdown.TokenType.LIST_ITEM_START) {
-        i++;
-        for (; tokens[i].type !== Markdown.TokenType.LIST_ITEM_END; i++) {
-          const token = tokens[i] as Markdown.TextToken;
+      if (currentVersion && currentSection && token.type === Markdown.TokenType.LIST) {
+        for (const item of token.items) {
+          const text =
+            item.tokens.find(
+              (token): token is Markdown.TextToken => token.type === Markdown.TokenType.TEXT
```

Comment:
Maybe it's feasible to extract this function the same way you have `isVersionToken`? 🤔

Reply:
I've just added a few type guards to `Markdown.ts`, `isTextToken` fits more there – `isVersionToken` is more specific for changelogs.